spec/requests/favourite_rooms_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe "FavouriteRooms", type: :request do
+  describe "PUT /favourite" do
```

Comment:
I think the route is now `/rooms/:id/favourite`

Reply:
Yes, these titles are just generally dumb. For instance, that for PUT `/rooms/:id` is called `PATCH /update` 🤷